test/fixtures/all-parameters-static/__snapshots__/all-parameters-static.spec.js.snap

```diff
@@ -1,3 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`all-parameters-static 1`] = `"const imageUrl = \`\${\\"https://res.cloudinary.com/trivago/image/upload/c_fill,h_300,q_auto:good,t_crop,w_200/\\"}\${\\"gallery/sand/\\"}\${\\"my-picture\\"}\${\\"_v10\\"}\${\\".png\\"}\`;"`;
+exports[`all-parameters-static 1`] = `"const imageUrl = \`\${\\"https://res.cloudinary.com/trivago/image/upload/c_fill,h_300,q_auto:good,t_crop,w_200/\\"}\${\\"gallery/sand/\\"}\${\\"my-picture\\"}\${\\"_v10\\"}\${\\".png\\"}\${\\"\\"}\`;"`;
```

Comment:
Seems like we now generate a bit of additional code. Is there any way to avoid doing that?

Reply:
I will investigate more on that, but to just explain why there is more code. The reason is that apparently by updating to babel 7, we have to make sure that the number of the quasis are exactly 1 more than the expressions. But I'll definitely dive into this 👍